src/core/ledger/currency.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import {
+  ethAddressParser,
+  type EthAddress,
+} from "../../plugins/ethereum/ethAddress";
+import * as C from "../../util/combo";
+
+/**
+ * EvmChainId is represented in number form for all EVM-based chains,
+ * including mainnet (1), and xDai (100)
```

Comment:
Docstring suggests it's represented in number form, but is actually a string? By "number form" do you mean "as a stringified number"?

Reply:
I went back and forth on how to describe this, and decided (wrongly) to let the code demonstrate what was meant. Thanks for the suggestion.